bin/cliain/src/main.rs

```diff
@@ -12,6 +12,10 @@ struct Config {
     #[clap(long, default_value = "127.0.0.1:9944")]
     pub node: String,
 
+    /// Whether to use `ws` or `wss` protocol
+    #[clap(long)]
```

Comment:
Can we make a default `false` here? Or it is what it's going to be when one does not pass `--ssl` arg. There're some scripts in https://github.com/Cardinal-Cryptography/aleph-node/pull/338 which do not use this `cliain` version yet.

Reply:
afaik it is `false` by default (actually I cannot set `default-value` for `bool`)

Reply:
Yes, it makes sense to be `false` by default, just wanted to confirm, thanks.